embedded/CMakePresets.json

```diff
@@ -0,0 +1,14 @@
+{
+  "version": 3,
+  "configurePresets": [
+    {
+      "name": "pico",
+      "hidden": false,
+      "generator": "Unix Makefiles",
+      "cacheVariables": {
+        "CMAKE_TOOLCHAIN_FILE": "$env{HOME}/pico-sdk/cmake/pico_arm_gcc.cmake",
+        "PICO_SDK_PATH": "$env{HOME}/pico-sdk"
```

Comment:
Are you sure if we need this file? Then add some description what this file does in the PR.

Also, I don't think this will work, it requires the pico-sdk to be in a specific directory.

If you really need this please add instructions.